shell/pkg/import.js

```diff
@@ -1,10 +1,10 @@
-const _NAME = require.context('BASE/NAME', true, /\.(vue|js|yaml)$/).keys();
+const _NAME = require.context('BASE/DIR', true, /\.(vue|js|yaml)$/).keys();
```

Comment:
![Screen Shot 2022-06-22 at 10 38 46 AM](https://user-images.githubusercontent.com/42977925/175107587-28aeb2c9-56a8-40b6-bc6f-8254215d9d42.png)
@nwmac here's the error thrown on hyphenated folder names. It looks like it's sufficient to clean up the string substituted for _NAME here so I've done that and reverted my renaming of cloud-credential and machine-config.